### PR TITLE
Junit servlet 3

### PR DESCRIPTION
**Pull request by akansha116**

### CODE REVIEW

File: core/src/test/java/com/hcl/ecomm/core/servlets/AddToWishListServletTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                "}";
+        return payload;
+    }
+
+    private String getSubmitResponse() {
+        String submitResponse = "{\n" +
+                "    \"message\": {\n" +
+                "        \"response\": \"true\"\n" +
+                "    },\n" +
+                "    \"status\": true\n" +
```
Comment (subrat-dash-hcl) [inline core/src/test/java/com/hcl/ecomm/core/servlets/AddToWishListServletTest.java]: change to \"statuscode\": 200\n"

File: core/src/test/java/com/hcl/ecomm/core/servlets/CountryStateServletTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+class CountryStateServletTest {
+    private CountryStateServlet countryStateServlet;
+    private ShippingInfoServiceImpl shippingInfoService;
+
+    @BeforeEach
+    protected  void setUp()  {
+        countryStateServlet = spy (new CountryStateServlet());
+    }
+
+    @Test
```
Comment (subrat-dash-hcl) [inline core/src/test/java/com/hcl/ecomm/core/servlets/CountryStateServletTest.java:35]: Write a test condition to check the else part to increase the code coverage

File: core/src/test/java/com/hcl/ecomm/core/servlets/CustomerChangePasswordServletTest.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    private CustomerChangePasswordServlet customerChangePasswordServlet;
+
+
+    @BeforeEach
+    protected void setUp() {
+        customerChangePasswordServlet = spy(new CustomerChangePasswordServlet());
+    }
+
+    @Test
```
Comment (subrat-dash-hcl) [inline core/src/test/java/com/hcl/ecomm/core/servlets/CustomerChangePasswordServletTest.java:36]: same as above , condition to check else part to increase code coverage

File: core/src/test/java/com/hcl/ecomm/core/servlets/AddToWishListServletTest.java (hunk 1 above)
Comment (akansha116) [reply core/src/test/java/com/hcl/ecomm/core/servlets/AddToWishListServletTest.java]: review comments incorporated

**Pull request merged**